host/xtest/xtest_main.c

```diff
@@ -78,26 +79,77 @@ ADBG_SUITE_ENTRY(XTEST_TEE_10001, NULL)
 ADBG_SUITE_ENTRY(XTEST_TEE_10002, NULL)
 ADBG_SUITE_DEFINE_END()
 
+
+ADBG_SUITE_DECLARE(XTEST_TEE_BENCHMARK)
+
+ADBG_SUITE_DEFINE_BEGIN(XTEST_TEE_BENCHMARK, NULL)
+ADBG_SUITE_ENTRY(XTEST_TEE_BENCHMARK_1001, NULL)
+ADBG_SUITE_ENTRY(XTEST_TEE_BENCHMARK_1002, NULL)
+ADBG_SUITE_ENTRY(XTEST_TEE_BENCHMARK_1003, NULL)
+ADBG_SUITE_ENTRY(XTEST_TEE_BENCHMARK_1004, NULL)
+ADBG_SUITE_ENTRY(XTEST_TEE_BENCHMARK_1005, NULL)
+ADBG_SUITE_ENTRY(XTEST_TEE_BENCHMARK_1006, NULL)
+ADBG_SUITE_DEFINE_END()
+
 static const char gdevname_tz[] = "opteearmtz00";
 char *_device = (char *)gdevname_tz;
 
+static const char gsuitename[] = "test";
+char *_suite = (char *)gsuitename;
```

Comment:
As per the C standard, identifiers with files scope starting with an underscore are reserved.